### ISSUE TITLE
Extra reminders for missed pending expenses

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#4747]
## User story 
As a Collective admin, I want to be reminded of pending expenses I need to approve if I missed the initial notification or an expense was edited and need re-approval.

## Best solution for this problem
A couple more notifications in certain situations.

If there are pending expenses *and* the Collective balance is enough to pay at lease one of them, in addition to the notification we already send out when the expense is first submitted, there's another notification on day 7 sent to the Collective admins.

> Subject: Expense pending on [Collective]: [Amount] for [exp title]

> This expense has been pending for 7 days. Please approve it if you would like it to be processed for payment.

Then paste in the content from the existing day 1 expense submitted notification.

## Priority

Metabase says there are 1,587 pending expenses right now.

Not sure if that's low or high so either P2 or P4.

**Comment by BenJam** [comment]
> Metabase says there are 1,587 pending expenses right now.

Looking into this as I had a related issue: discounting the current month it looks like we have something in the realm of 4% pending expenses over 30 days old, 3% over 60 days, 2% over 90 days and it drops from there. 

For me this kinda seems like an acceptable level so I'd say this is quite. alow priority.

**Comment by znarf** [comment]
A quick thing could be to always display Pending Expenses first for Collective admins, so they don't end up lost in the list.

**Comment by BenJam** [comment]
@poohlaga how would you feel about this solution? Wondering about including this as a quick fix but it would require that we run though all pending expenses and reject those that are out of date.

**Comment by poohlaga** [comment]
I don't think it is needed anymore. Seems like this was opened before the rules were put in place for needing receipts which I am sure was a lot of the reason reminders were needed. IMHO close this one but ping @alanna unless there is a use case I am missing.

**Comment by alanna** [comment]
This isn't about Virtual Cards @poohlaga - it's about pending expenses

**Comment by alanna** [comment]
For example at Gift Collective, we regularly go through our Collectives' pending expenses and find some every week where the admin just forgot to hit the approve button so they're not getting paid. Especially if we're focusing on supporting Collectives to use their money, I think this is an easy and worthwhile improvement.

**Comment by natehn** [comment]
Ah, @poohlaga and I were thinking this was for hosts - but I understand, this is for Collective admins. Makes sense.
